tests/test_plugin_detection.py

```diff
@@ -41,6 +40,11 @@
         # Mock appears to not be installed
         could_import_mock = False
 
+try:
+    # Python 3.4 forwards use importlib.reload
```

Comment:
```suggestion
    # Python3:  use importlib.reload
```
very small nitpick to make it easier to find this later and remove it (when we eventually drop python2 support).